client/galaxy/scripts/nls/fr/locale.js

```diff
@@ -0,0 +1,272 @@
+"View data" :
+    "Voir les données",
+// download button
+"Download" :
+    "Télécharger",
+"Download dataset" :
+    "Télécharger le jeu de données",
+"Additional files" :
+    "Fichiers additionnels",
+// info/show_params
+"View details" :
+    "Voir les détails",
+
+// dataset states
+// state: new
+"This is a new dataset and not all of its data are available yet" :
+    "Il s'agit d'un nouveau jeu de données et seule une partie des données est accessible pour le moment",
+// state: noPermission
+"You do not have permission to view this dataset" :
+    "Vous n'avez pas la permission de voir ce jeu de données",
+// state: discarded
+"The job creating this dataset was cancelled before completion" :
+    "Le calcul ayant créé ce jeu de données a été annulé prématurément",
+// state: queued
+"This job is waiting to run" :
+    "Ce calcul est en attente de traitement",
+// state: upload
+"This dataset is currently uploading" :
+    "Ce jeu de données est en cours de téléversement",
+// state: setting_metadata
+"Metadata is being auto-detected" :
+    "Les métadonnées sont auto-détectées",
+// state: running
+"This job is currently running" :
+    "Le traitement est en cours",
+// state: paused
+"This job is paused. Use the \"Resume Paused Jobs\" in the history menu to resume" :
+    "Ce traitement est en pause. Utilisez le \"Relancer les traitements en pause\" dans le menu d'historique pour le relancer",
+// state: error
+"An error occurred with this dataset" :
+    "Un erreur est survenue avec ce jeu de données",
+// state: empty
+"No data" :
+    "Aucune donnée",
+// state: failed_metadata
+"An error occurred setting the metadata for this dataset" :
+    "Une erreur est survenue pendant la récupération des métadonnées de ce jeu de données",
+
+// ajax error prefix
+"There was an error getting the data for this dataset" :
+    "Il est survenu une erreur durant la récupération du contenu de ce jeu de données",
+
+// purged'd/del'd msg
+"This dataset has been deleted and removed from disk" :
+    "Ce jeu de données a été supprimé et effacé du disque",
+"This dataset has been deleted" :
+    "Ce jeu de données a été supprimé",
+"This dataset has been hidden" :
+    "Ce jeu de données a été caché",
+
+"format" :
+    "format",
+"database" :
+    "génome de référence",
+
+// ---- hda-edit
+"Edit attributes" :
+    "Editer les attributs",
+"Cannot edit attributes of datasets removed from disk" :
+    "Impossible d'éditer les attributs de jeux de données effacés du disque",
+"Undelete dataset to edit attributes" :
+    "Restaurer le jeu de données pour en éditer les attributs",
+"This dataset must finish uploading before it can be edited" :
+    "Ce jeu de données doit être entièrement téléversé avant avant toute modification",
+"This dataset is not yet editable" :
+    "Ce jeu de données n'est pas encore éditable",
+
+"Delete" :
+    "Supprimer",
+"Dataset is already deleted" :
+    "Pov' tâche, le jeu de données est déjà supprimé ;)",
```

Comment:
Well, it translates more or less to `dumbass, the dataset is already deleted`. We could be a bit friendlier here ;),
`"Le jeu de données est déjà supprimé ;)"` would be fine.